src/test/java/org/gridsuite/actions/server/ContingencyListControllerTest.java

```diff
@@ -364,6 +375,7 @@ public void emptyScriptTest() throws Exception {
                 .andExpect(status().isOk())
                 .andExpect(content().contentTypeCompatibleWith(APPLICATION_JSON))
                 .andExpect(content().json("[]"));
+        cleanDB();
```

Comment:
useless because it's already in the setup

Reply:
Done